postgraphiql/src/components/PostGraphiQL.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import GraphiQL from 'graphiql';
-import { getOperationAST, parse } from 'graphql';
+import {buildClientSchema, getIntrospectionQuery, getOperationAST, GraphQLObjectType, isType, parse} from 'graphql';
```

Comment:
I do not understand why prettier is not formatting this file in the way I'd expect. It doesn't seem to be in the `prettierignore` file, and it doesn't seem to be excluded from the prettier glob

https://github.com/graphile/postgraphile/blob/55bff41460b113481c8161ef8f178f5af0a17df3/package.json#L48

And yet, CI passes. Very strange. I will have to investigate at some point.